artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -4913,6 +4913,14 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="security-caches" type="xsd:boolean" default="false" maxOccurs="1" minOccurs="0">
```

Comment:
shouldn't this be called security-events, perhaps security-attempts?

Reply:
The instances of Caffeine caches which are related to security is what is being instrumented here. Metrics that are made available when this is set to `true` are things like:

- `cache.size`
- `cache.puts`
- `cache.gets` tagged by `result` - either `hit` or `miss`
- `cache.evictions`
- `cache.eviction.weight`

Therefore, calling it `security-caches` seems appropriate.